[2023-10-15T19:22:30.081+0300] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag_all_models_in_one.task_get_data scheduled__2023-10-14T01:00:00+00:00 [queued]>
[2023-10-15T19:22:30.085+0300] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag_all_models_in_one.task_get_data scheduled__2023-10-14T01:00:00+00:00 [queued]>
[2023-10-15T19:22:30.085+0300] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-15T19:22:30.145+0300] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): task_get_data> on 2023-10-14 01:00:00+00:00
[2023-10-15T19:22:30.149+0300] {standard_task_runner.py:57} INFO - Started process 11560 to run task
[2023-10-15T19:22:30.151+0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'test_dag_all_models_in_one', 'task_get_data', 'scheduled__2023-10-14T01:00:00+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_several_parallel_model_train.py', '--cfg-path', '/tmp/tmp6siv5lap']
[2023-10-15T19:22:30.153+0300] {standard_task_runner.py:85} INFO - Job 87: Subtask task_get_data
[2023-10-15T19:22:30.199+0300] {task_command.py:415} INFO - Running <TaskInstance: test_dag_all_models_in_one.task_get_data scheduled__2023-10-14T01:00:00+00:00 [running]> on host dsbulgako1-QEMU-Virtual-Machine
[2023-10-15T19:22:30.254+0300] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='dmitrii.bulghakov@gmail.com' AIRFLOW_CTX_DAG_OWNER='Dmitry Bulgakov' AIRFLOW_CTX_DAG_ID='test_dag_all_models_in_one' AIRFLOW_CTX_TASK_ID='task_get_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-14T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-14T01:00:00+00:00'
[2023-10-15T19:22:30.265+0300] {test_dag_several_parallel_model_train.py:56} INFO - Postgres connection initializing.
[2023-10-15T19:22:30.274+0300] {base.py:73} INFO - Using connection ID 'pg_connection' for task execution.
[2023-10-15T19:22:30.281+0300] {test_dag_several_parallel_model_train.py:61} INFO - Taking data from Postgres.
[2023-10-15T19:22:30.285+0300] {logging_mixin.py:151} WARNING - /home/dsbulgako1/my_repos/mlops_project/dags/test_dag_several_parallel_model_train.py:62 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2023-10-15T19:22:30.345+0300] {test_dag_several_parallel_model_train.py:64} INFO - S3 connection initializing.
[2023-10-15T19:22:30.353+0300] {base.py:73} INFO - Using connection ID 's3_connector' for task execution.
[2023-10-15T19:22:30.354+0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='s3_connector', conn_type='aws') credentials retrieved from login and password.
[2023-10-15T19:22:30.391+0300] {configprovider.py:949} INFO - Found endpoint for s3 via: config_global.
[2023-10-15T19:22:30.396+0300] {test_dag_several_parallel_model_train.py:69} INFO - S3 data uploading.
[2023-10-15T19:22:31.174+0300] {test_dag_several_parallel_model_train.py:72} INFO - Data uploading finished.
[2023-10-15T19:22:31.177+0300] {python.py:194} INFO - Done. Returned value was: {'model_rf': 'rf', 'model_lr': 'lr', 'model_hgb': 'hgb', 'start_timestamp': '20231015 19:22', 'data_uploaded_timestamp': '20231015 19:22'}
[2023-10-15T19:22:31.220+0300] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=test_dag_all_models_in_one, task_id=task_get_data, execution_date=20231014T010000, start_date=20231015T162230, end_date=20231015T162231
[2023-10-15T19:22:31.258+0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-15T19:22:31.279+0300] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
